modules/manage/pages/schema-reg/schema-reg-authorization.adoc

```diff
@@ -0,0 +1,355 @@
+
+[,bash]
+----
+rpk security acl create --allow-principal panda --operation read --topic bar --registry-subject bar-value
+
+PRINCIPAL   HOST  RESOURCE-TYPE  RESOURCE-NAME  RESOURCE-PATTERN-TYPE  OPERATION  PERMISSION  ERROR
+User:panda  *     SUBJECT        bar-value      LITERAL                READ       ALLOW
+User:panda  *     TOPIC          bar            LITERAL                READ       ALLOW
+
+----
+
+=== Create an ACL for global Schema Registry access
+
+This example grants the user `jane` global read and write access to the Schema Registry, plus read and write access to the topic `private`. The `--registry-global` flag creates ACLs for all global Schema Registry operations.
+
+[,bash]
+----
+rpk security acl create --allow-principal jane --operation read,write --topic private --registry-global
+PRINCIPAL  HOST  RESOURCE-TYPE  RESOURCE-NAME  RESOURCE-PATTERN-TYPE  OPERATION  PERMISSION  ERROR
+User:jane  *     REGISTRY                      LITERAL                READ       ALLOW
+User:jane  *     REGISTRY                      LITERAL                WRITE      ALLOW
+User:jane  *     TOPIC          private        LITERAL                READ       ALLOW
+User:jane  *     TOPIC          private        LITERAL                WRITE      ALLOW
+----
+
+User `jane` now has global `read` and `write` access to the Schema Registry, and to the topic `private`.
+
+=== Create a role with Schema Registry ACLs
+
+You can combine Schema Registry ACLs with xref:manage:security/authorization/rbac[Role-Based Access Control (RBAC)] to create reusable roles. This approach simplifies permission management when you need to assign the same set of permissions to multiple users.
+
+This example creates a role called `SoftwareEng` and assigns it ACLs for both topic and Schema Registry access:
+
+[NOTE]
+====
+Redpanda recommends using the topic naming strategy for Schema Registry subjects, where subjects follow the pattern `<topicName>-key` or `<topicName>-value`. For details, see xref:manage:schema-reg/schema-id-validation#set-subject-name-strategy-per-topic.adoc[Set subject name strategy per topic].
+====
+
+[,bash]
+----
+# Create the role
+rpk security role create SoftwareEng
+
+# Create ACLs for the role
+rpk security acl create \
+  --operation read,write \
+  --topic private \
+  --registry-subject private-key,private-value \
+  --allow-role SoftwareEng
+
+# You can add more ACLs to this role later
+rpk security acl create --allow-role "RedpandaRole:SoftwareEng" [additional-acl-flags]
+----
+
+After creating the role, assign it to users:
+
+[,bash]
+----
+
+rpk security role assign SoftwareEng --principal User:john,User:jane
+Successfully assigned role "SoftwareEng" to
+NAME  PRINCIPAL-TYPE
+john  User
+jane  User
+----
+
+=== Troubleshooting ACL creation
+
+When creating ACLs that include Schema Registry subjects, you might encounter errors if the subject doesn't exist or if there are configuration issues.
+
+==== Common error: Subject not found
```

Comment:
Redpanda should be able to create ACLs for non-existing subjects.

The `Not found` error you are seeing here is the request to `12:17:33.935 DEBUG sending request {"method": "POST", "URL: "http://127.0.0.1:8081/security/acls", "has_bearer": false, "has_basic_auth": false}` 

returned a `Not found`, meaning that the endpoint is not available (i.e you are using an old Redpanda version)